infrastructure/Dockerfile

```diff
@@ -16,6 +16,8 @@ COPY target/bin /usr/share/corfu/bin
 COPY target/corfu_scripts /usr/share/corfu/corfu_scripts
 COPY target/scripts /usr/share/corfu/scripts
 COPY target/logback.prod.xml /usr/share/corfu/conf/logback.prod.xml
+COPY target/corfu-compactor-config.yml /usr/share/corfu/conf/corfu-compactor-config.yml
+COPY target/compactor-logback.prod.xml /usr/share/corfu/conf/compactor-logback.prod.xml
```

Comment:
I think there might be an issue with copying it to `/usr/share/corfu/conf`.
Currently `/usr/share/corfu/conf` is mounted from a configmap called `{{ include "corfu.fullname" . }}-corfu-configs`
https://github.com/CorfuDB/corfudb-cloud/blob/master/cloud/corfu/corfu/templates/statefulset.yaml#L95
It's usage is defined here:
https://github.com/CorfuDB/corfudb-cloud/blob/master/cloud/corfu/corfu/templates/statefulset.yaml#L110
And here is the configmap definition itself:
https://github.com/CorfuDB/corfudb-cloud/blob/master/cloud/corfu/corfu/templates/configmap.yaml
As you can see it includes all the files from here:
https://github.com/CorfuDB/corfudb-cloud/tree/master/cloud/corfu/corfu/config
On SaaS (prod) we also enhance this directory with some other stuff for LR for example. 
The solution is to either copy it to the different dir within the image itself or place these yaml files in this dir: 
https://github.com/CorfuDB/corfudb-cloud/tree/master/cloud/corfu/corfu/config
so they are mounted as a configmap.

Reply:
perfect! thank you - I think they are both needed right?